### DISCUSSION TITLE
[Suggestion] PiVPN Web UI

### DISCUSSION THREAD

**Discussion by randomap** [pivpn/pivpn#1529] [Feature requests]
### Has your install failed? 
no

### Describe the issue
I would like to manage the clients in PiVPN through a Web UI. Also can there be a rename clients command to manage and organize easier. 

#### What type of hardware are you running PiVPN at?
Raspberry Pi 0 WH

#### I dont know where to put feedback or suggestions and I hope this is useful. Thanks for the great app.

**Comment by coolapso** [comment]
Moving to Discussions on feature requests. 

This is actually that we would like to have, and has been discussed multiple times (this was the latest one: #1258 ) and several ppl tried to make their own, however they always lacked the support for some pivpn features or ppl did not want to commit to the project and stick around to support whenever problems come up. 

Unfortunately, I completely lack the knowledge and time to develop a front end page for PiVPN, I suspect the same might happen with @orazioedoardo 

I am glad to accept suggestions and pull requests for this and give a contributor status to whoever wants to implement this and stick around to support the project when issues come up.

**Comment by orazioedoardo** [comment]
> Unfortunately, I completely lack the knowledge and time to develop a front end page for PiVPN, I suspect the same might happen with @orazioedoardo

Mostly boils down to time, a web UI would be really nice to have though.
